specification/validated-relationships-service-api.yaml

```diff
@@ -280,6 +281,81 @@ paths:
                 internalServerError:
                   $ref: '#/components/examples/InternalServerError'
 
+  /Consent:
+    get:
+        summary: Get proxy relationships
+        description: |
+          ## Overview
+          Use this endpoint to get the details about proxy relationships, including current status, based on a provided proxy's NHS Number.
```

Comment:
Above for GET /RelatedPerson you've used `Use this endpoint to get relationships that have been verified against authoritative sources for a proxy with a given NHS Number.` where as for this endpoint it says `get the details about proxy relationships`. Is this endpoint not using authoritative sources? 

Presuming that both endpoints are using `authoritative sources` would it best to remove the phrase `authoritative sources` ? As to me it suggests this endpoint's data is less reliable

Reply:
good point. I'll see about some better language. The point I want to make clear is that one is proxy relationships, one is candidate relationships